static/css/style.css

```diff
@@ -17,12 +17,22 @@ body {
 }
 
 /* Set the fixed height of the footer here */
-#push,
-#footer {
-	height: 60px;
+#footrow{
```

Comment:
could you use more descriptive class names?
and please use spaces after selectors, look at other selectors in the file